server/mdm/apple/cert.go

```diff
@@ -80,9 +81,8 @@ func GetSignedAPNSCSR(client *http.Client, csr *x509.CertificateRequest) error {
 	// for testing
 	baseURL := defaultFleetDMAPIURL
 	if x := os.Getenv("TEST_FLEETDM_API_URL"); x != "" {
-		baseURL = x
+		baseURL = strings.TrimRight(x, "/")
```

Comment:
nit: should we do `path.Join` below instead?

Reply:
Given that it's just for a test override and it handles the one case that we care about (if the override URL ends with a slash and we combine with a path that starts with one), using `url.Parse` with its error handling + `path.Join` on the `Path` field seemed overkill to me.

Reply:
ah, right thank you! I forgot we also have to parse the URL to do the join 😅